client/components/Shop/Products/Products.jsx

```diff
@@ -46,13 +46,13 @@ const ProductList = ({ currentCategory }) => {
     object-fit: cover;
 `;
 //   // from loungeo
```

Comment:
Looks like this works, but I have a question - why is this from loungeo? This looks like a simple styled div.

Reply:
@jessicaawhy  This is just temporarily from there as we are just testing things out.